cfme/infrastructure/virtual_machines.py

```diff
@@ -247,8 +247,9 @@ def migrate_vm(self, email=None, first_name=None, last_name=None,
             "last_name": last_name,
             "email": email,
             "host_name": {"name": prov_data.get("host")},
-            "datastore_name": {"name": prov_data.get("datastore")}
         }
+        if not self.provider.type == 'rhevm':
```

Comment:
@REQUIRED: please use correct constructions like != instead of not ==
readability counts (c)